---
title: C28125
ms.date: 11/04/2016
ms.prod: visual-studio-dev15
ms.technology: vs-ide-code-analysis
ms.topic: reference
f1_keywords:
- C28125
helpviewer_keywords:
- C28125
ms.assetid: 3f7b8db4-6a64-4480-919e-3f3ceca7f948
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: 3bd143513dfde9e6a1392ed0f88ae246e4766ae1
ms.sourcegitcommit: 1ab675a872848c81a44d6b4bd3a49958fe673c56
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 09/10/2018
ms.locfileid: "44283503"
---
# <a name="c28125"></a>C28125
Aviso C28125: A função deve ser chamada de dentro de um bloco try / except bloco

 O driver está chamando uma função que deve ser chamada de dentro de um bloco try / except bloco, tais como [ProbeForRead](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforread), [ProbeForWrite](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforwrite), [MmProbeAndLockPages](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-mmprobeandlockpages).

## <a name="example"></a>Exemplo
 O exemplo de código a seguir gera este aviso:

```
ProbeForRead(addr, len, 4);
```

 O exemplo de código a seguir evita este aviso:

```

__try
{
    ProbeForRead(addr, len, 4);
}
__except(EXCEPTION_EXECUTE_HANDLER)
{
    Status = GetExceptionCode();
    ... report error status
}
```